content/en/tracing/guide/monitor-kafka-queues.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Tracing Kafka queues
+kind: guide
+further_reading:
+- link: "/tracing/trace_collection"
+  tags: "Documentation"
+  text: "Set up trace collection"
+- link: "/integrations/kafka"
+  tags: "Documentation"
+  text: "Kafka integration"
+- link: "/tracing/troubleshooting/"
+  tags: "Documentation"
+  text: "Data Streams Monitoring"
+---
+
+## Monitor Kafka queues
+
+In event-driven pipelines, queuing and streaming technologies such as Kafka are essential to the successful operation of your systems. However, ensuring that messages are being reliably and quickly conveyed between services is difficult due to the many technologies and teams involved in any such environment. Datadog's Kafka integration and its APM suite enables your team to monitor your infrastructure and pipelines health and efficiency.
+
+### Integration
+
+Easily visualize the performance of your cluster in real time and correlate the performance of Kafka with the rest of your applications by using [Datadog's Kafka integration][1]. Datadog also provide a [MSK integration][2] if need be.
+
+{{< img src="tracing/guide/monitor_kafka_queues/kafka_dashboard.png" alt="Data Streams Monitoring Demo">}}
+<br>
+
+### Data Stream Monitoring
+
+[Datadog Data Streams Monitoring][3] provides a standardized method for your teams to measure pipeline health and end-to-end latencies for events traversing across your system. The deep visibility offered by Data Streams Monitoring enables you to pinpoint faulty producers, consumers, or queues driving delays and lag in the pipeline; discover hard-to-debug pipeline issues such as blocked messages, hot partitions, or offline consumers; and collaborate seamlessly across relevant infrastructure or app teams.
+
+{{< img src="tracing/guide/monitor_kafka_queues/dash-2022-data-streams-compressed-blurb.mp4" alt="Data Streams Monitoring Demo" video="true">}}
+<br>
+
+### Collect distributed traces
+
+Datadog APM and distributed tracing gives you expanded visibility into the performance of your services by measuring request volume and latency. You create graphs and alerts to monitor your APM data, and you can visualize the activity of a single request in a flame graph like the one shown below to better understand the sources of latency and errors.
+<br>
+{{< img src="tracing/guide/monitor_kafka_queues//kafka_trace.png" alt="A kafka consumer span" >}}
+<br>
+Datadog APM can automatically trace requests to and from Kafka clients. This means you can collect traces without modifying your source code. Note that we inject headers in the kafka messages to be able to propagate the context of the trace from the producer to the consumer.
+
+#### Setup
```

Comment:
It would be good to add a note to the compatibility section to check which Kafka libraries for the different languages are supported: https://docs.datadoghq.com/tracing/trace_collection/compatibility/ .

Reply:
Thanks fixed in 3d6f82681d0a0ffb88aaca7598dba50c3b9ba0b8